vllm/v1/attention/backends/flashinfer.py

```diff
@@ -154,6 +154,10 @@ def get_supported_head_sizes(cls) -> list[int]:
         # https://github.com/flashinfer-ai/flashinfer/blob/3d55c71a62052c590c130897d3a3db49b14fcc34/include/flashinfer/utils.cuh#L157
         return [64, 128, 256]
 
+    @staticmethod
+    def get_supported_block_size() -> list[Union[int, MultipleOf]]:
+        return [MultipleOf(1)]
```

Comment:
I think we need to take into account here whether FlashInfer is using TRTLLM (it has two different paths). When TRTLLM is enabled there are additional constraints on the block size (it can't be larger than 128).